test/integration/api/IssuanceAPI.spec.ts

```diff
@@ -229,25 +229,26 @@ describe('IssuanceAPI', () => {
 
     describe('when the caller does not have enough of a component', async () => {
       let componentWithInsufficientBalance: StandardTokenMockContract;
+      let userTokenBalance: BigNumber;
 
       beforeEach(async () => {
-        // Only the first component will have a large unit
-        componentUnits = _.map(componentTokens, (token, index) => {
-          if (index > 0) {
-            return new BigNumber(1);
-          }
-          return setComponentUnit;
-        });
+        // Only the first component will have an insufficient balance
+        componentUnits = _.map(componentTokens, (token, index) =>
+          index === 0 ? new BigNumber(1) : setComponentUnit);
```

Comment:
This still came up flaky for me locally

Reply:
Will fix it in a separate pr since I introduced it originally, but yeah the logic for this was suboptimal @felix2feng